conans/client/configure_build_environment.py

```diff
@@ -160,7 +160,7 @@ def configure(self, configure_dir=None, args=None, build=None, host=None, target
                 triplet_args.append("--target %s" % (target or auto_target))
 
         with environment_append(self.vars):
-            self._conanfile.run("%sconfigure %s %s" % (configure_dir, args_to_string(args), " ".join(triplet_args)))
+            self._conanfile.run("%s/configure %s %s" % (configure_dir, args_to_string(args), " ".join(triplet_args)))
```

Comment:
Maybe check if current "configure_dir" contains a final "/" and remove it to avoid double slash.